test/common/testlib.py

```diff
@@ -2090,6 +2090,19 @@ def write_file(self, path: str, content: str, append: bool = False, owner: Optio
         m = self.machine
         self.restore_file(path, post_restore_action=post_restore_action)
         m.write(path, content, append=append, owner=owner, perm=perm)
+        
+    
+    def reboot(self, timeout_sec:Optional[int] = None):
+        self.allow_restart_journal_messages()
+        if timeout_sec is None:
+            self.machine.reboot()
+        else:
+            self.machine.reboot(timeout_sec=timeout_sec)
+        
+
+    def wait_reboot(self, timeout_sec):
+        self.allow_restart_journal_messages()
+        self.machine.wait_reboot(timeout_sec=timeout_sec)
```

Comment:
This needs the same `if` treatment, or changing bots first to do the right thing on `None`.